config/F2.config

```diff
@@ -33,6 +33,16 @@ process {
             }
         }
     }
+    withName: assess_coverage_mosdepth {
+        cpus = 1
+        memory = 1500.MB
```

Comment:
**(question|non-blocking)**

Can Nextflow take `1.5.GB`? If so, minor point but I think using the same unit across configs would make more sense.